doc/testing.html

```diff
@@ -592,18 +589,14 @@ <h3 id="pkcs11-tests">PKCS11 Tests</h3>
     JTREG=&quot;JAVA_OPTIONS=-Djdk.test.lib.artifacts.nsslib-linux_aarch64=/path/to/NSS-libs&quot;</code></pre>
 <p>For more notes about the PKCS11 tests, please refer to
 test/jdk/sun/security/pkcs11/README.</p>
-<h3 id="testing-with-external-security-providers">Testing with external
-security providers</h3>
-<p>Some security tests currently use a hardcoded provider for
-KeyFactory, Cipher, KeyPairGenerator, KeyGenerator, or SecretKeyFactory.
-Specify the <code>-Dtest.provider.name=NAME</code> property, to use a
-specific provider when instantiating the service.</p>
-<h3 id="testing-with-external-security-providers-1">Testing with
-external security providers</h3>
-<p>Some security tests currently use a hardcoded provider for
-KeyFactory, Cipher, KeyPairGenerator, KeyGenerator, or SecretKeyFactory.
-Specify the <code>-Dtest.provider.name=NAME</code> property, to use a
-specific provider when instantiating the service.</p>
+<h3 id="testing-with-alternative-security-providers">Testing with
+alternative security providers</h3>
+<p>Some security tests use a hardcoded provider for
+<code>KeyFactory</code>, <code>Cipher</code>,
+<code>KeyPairGenerator</code>, <code>KeyGenerator</code>, or
+<code>SecretKeyFactory</code> objects. Specify the
```

Comment:
I see some tests use a hard-coded provider for `Signature` too. Can you double-check all the changes and add any that you missed?